indico/modules/categories/__init__.py

```diff
@@ -41,6 +42,12 @@ def _merge_users(target, source, **kwargs):
     CategoryPrincipal.merge_users(target, source, 'category')
 
 
+def _is_moderation_visible(category):
+    return category.event_requires_approval or category.event_move_requests.filter(
+        EventMoveRequest.state == MoveRequestState.pending).first() or any(
+        ['event_move_request' in e.permissions for e in category.acl_entries])
```

Comment:
```suggestion
    return (
        category.event_requires_approval or
        category.event_move_requests.filter_by(state=MoveRequestState.pending).has_rows() or
        any('event_move_request' in entry.permissions for entry in category.acl_entries)
    )
```

Much more readable IMO